src/OmniSharp.DotNetTest/VSTestManager.cs

```diff
@@ -26,8 +26,13 @@ public VSTestManager(Project project, string workingDirectory, IDotNetCliService
         {
         }
 
-        private object GetDefaultRunSettings(string targetFrameworkVersion)
+        private object LoadRunSettingsOrDefault(string runSettingsPath, string targetFrameworkVersion)
         {
+            if (runSettingsPath != null)
+            {
+                return File.ReadAllText(runSettingsPath);
```

Comment:
you should check if the file exists, if not, I'd fallback to defaults to not crash?

Reply:
Added an error message if runsettings load fails, and it will continue with default settings.